src/Control/Scrobbler/Lastfm.hs

```diff
@@ -78,7 +75,7 @@ scrobble Credentials { secret, apiKey, sessionKey } =
           , toList tss^..ifolded.indices (`elem` is)
           )
 
-  server    = maybe False (`elem` [11, 16]) . preview (key "error" . _Number)
+  server = maybe False (`elem` [11, 16]) . preview (key "error" . _Number)
```

Comment:
Maybe name these magic constants? Apparently, it was a mistake not to name them earlier.